### PR TITLE
Style & Setup Poses Screen

### PR DESCRIPTION
**Pull request by EmilioLombana**

### CODE REVIEW

File: client/js/App.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { UserProvider } from "./context/UserContext";
 import AppNavigator from "./navigation/RootStackNavigator";
```
Comment (siddharthparmar7) [inline client/js/App.js:5]: remove unnecessary imports.

File: client/js/screens/Poses/Poses.js [hunk 2]
```diff
@@ -1,18 +1,55 @@
...
-const Poses = ({ navigation }) => {
+const thumbnail = {
+  Bear: require("../../assets/images/icons/Bear.png"),
+  Superbug: require("../../assets/images/icons/Superbug.png")
+};
+
+renderSeparator = () => {
+  return (
+    <View
+      style={{
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/Poses.js]: all the styles should be in your styles.js file

File: client/js/screens/Poses/Poses.js [hunk 3]
```diff
@@ -1,18 +1,55 @@
...
+        height: 1,
+        backgroundColor: "#CED0CE"
+      }}
+    />
+  );
+};
+
+const Poses = ({ navigation, data }) => {
+  _keyExtractor = item => item.id;
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/Poses.js]: why _? did you mean to create a const?

File: client/js/screens/Poses/Poses.js [hunk 4]
```diff
@@ -1,18 +1,55 @@
...
+        ItemSeparatorComponent={this.renderSeparator}
+        renderItem={({ item }) => (
+          <View>
+            <TouchableOpacity
+              style={styles.Pose}
+              onPress={() => {
+                navigation.navigate("PosesDetails");
+              }}
+            >
+              {console.log(item.icon)}
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/Poses.js]: remove all the console logs.

File: client/js/screens/Poses/Poses.js [hunk 5]
```diff
@@ -1,18 +1,55 @@
...
+              <Image
+                source={thumbnail[item.icon]}
+                style={styles.image}
+              />
+              {console.log(item.icon)}
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/Poses.js]: remove this console log

File: client/js/screens/Poses/PosesContainer.js [hunk 6]
```diff
@@ -11,7 +25,19 @@ class PosesContainer extends Component {
     }
   };
   render() {
-    return <Poses navigation={this.props.navigation} />;
+    return (
+      <Query query={PosesQuery}>
+        {({ loading, error, data }) => {
+          if (loading)
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/PosesContainer.js]: what if there is an error?

File: client/js/screens/Poses/styles.js [hunk 7]
```diff
@@ -1,12 +1,30 @@
...
+    height,
+    justifyContent: "center"
+  },
+  image: {
+    position: "absolute",
+    left: 30
+  },
+  arrow: {
+    position: "absolute",
+    right: 20
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/styles.js]: we do not use left and right properties. Please use flexbox.

File: client/js/screens/Poses/Poses.js [hunk 8]
```diff
@@ -1,18 +1,55 @@
 import React from "react";
-import { Text, TouchableOpacity } from "react-native";
+import { View, Text, TouchableOpacity, FlatList, Image } from "react-native";
 import propTypes from "prop-types";
+import styles from "./styles";
 
-const Poses = ({ navigation }) => {
+const thumbnail = {
+  Bear: require("../../assets/images/icons/Bear.png"),
+  Superbug: require("../../assets/images/icons/Superbug.png")
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/Poses.js]: this is a good way to do it. Can you put all of your images here for this file?

File: client/js/screens/Poses/Poses.js [hunk 9]
```diff
@@ -1,18 +1,55 @@
...
+                source={require("../../assets/images/blackarrow.png")}
+                style={styles.arrow}
+              />
+            </TouchableOpacity>
+          </View>
+        )}
+      />
     </React.Fragment>
   );
 };
```
Comment (siddharthparmar7) [inline client/js/screens/Poses/Poses.js:53]: prop type validation for data?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you

**Review by 10thfloor — APPROVED** [review]

**Comment by 10thfloor** [comment]
CLosing this. Not sure what happened here.

**Pull request closed without merging**